flake-modules/dev/list-plugins.py

```diff
@@ -11,10 +11,38 @@
 
 EXCLUDES: list[str] = [
     # Not plugin files
```

Comment:
Maybe in the future we can detect plugins files? Or maybe that's non-trivial...

We could use the same strategy as the "by-name" and "package option" tests, i.e. scan the toplevel option set recursively, collect all options, then filter for `enable` options.

The option itself contains info on where it was declared, so we get the filename that way.

Obviously beyond this PR, and would need some nix code to complement the python.